packages/gp2-components/src/templates/DashboardPageBody.tsx

```diff
@@ -1,16 +1,27 @@
 import { ComponentProps } from 'react';
 import { css } from '@emotion/react';
 
-import { PagesSection, pixels, NewsSection } from '@asap-hub/react-components';
+import {
+  PagesSection,
+  pixels,
+  NewsSection,
+  Link,
+} from '@asap-hub/react-components';
+import { news as newsRoute } from '@asap-hub/routing';
 
-const { rem } = pixels;
+const { rem, perRem } = pixels;
 
 const styles = css({
   display: 'grid',
   gridRowGap: `${rem(72)}`,
   marginBottom: `${rem(24)}`,
 });
 
+const viewAllStyles = css({
+  marginTop: `${24 / perRem}em`,
```

Comment:
we can use the `rem` funtion and make it: 
```marginTop: rem(24),```

Reply:
I wonder if that function name should be pxToRem or something.